src/postprocess/phonon.jl

```diff
@@ -36,3 +99,24 @@ function compute_dynmat_cart(basis::PlaneWaveBasis, ψ, occupation; kwargs...)
     dynmats_reduced = compute_dynmat(basis, ψ, occupation; kwargs...)
     dynmat_red_to_cart(basis.model, dynmats_reduced)
 end
+
+function compute_dynmat(scfres::NamedTuple; kwargs...)
+    compute_dynmat(scfres.basis, scfres.ψ, scfres.occupation; scfres.ρ, scfres.ham,
+                   scfres.occupation_threshold, scfres.εF, scfres.eigenvalues, kwargs...)
+end
+
+function compute_dynmat_cart(scfres; kwargs...)
+    compute_dynmat_cart(scfres.basis, scfres.ψ, scfres.occupation; scfres.ρ, scfres.ham,
+                        scfres.occupation_threshold, scfres.εF, scfres.eigenvalues, kwargs...)
+end
+
+"""
+Assemble the right-hand side term for the Sternheimer equation for all relevant quantities.
```

Comment:
All relevant quantities for what ? a bit vague since this is phonon specific, right?

Reply:
I feel this function as well as `compute_δHψ` need clearer docstrings what the interface is, what is actually computed and returned, what ar the kwargs etc. Otherwise it is a bit magical.